df.world.xml

```diff
@@ -270,31 +270,15 @@
         <int32_t name='year_tick'/>
         <int32_t name='unk_4'/>
 
-        materials ?
-        <stl-vector name='unk_5' type-name='int16_t'/>
-        <stl-vector name='unk_6' type-name='int16_t'/>
-        <stl-vector name='unk_7' type-name='int16_t'/>
-        <stl-vector name='unk_8' type-name='int32_t'/>
-        <stl-vector name='unk_9' type-name='int32_t'/>
-
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
+        <stl-vector name='item_type' type-name='int16_t'/>
```

Comment:
ref-target attributes would be nice where applicable

Reply:
Heh, didn't mean to make it that complicated, but +1

Reply:
I just copied them from somewhere:)